spyder_kernels/console/kernel.py

```diff
@@ -252,10 +252,11 @@ def load_data(self, filename, ext):
         if error_message:
             return error_message
 
-        for key in list(data.keys()):
-            new_key = fix_reference_name(key, blacklist=list(glbs.keys()))
-            if new_key != key:
-                data[new_key] = data.pop(key)
+        if not override:
```

Comment:
Could you please add a comment explaining what this is actually doing now?

It is not so clear to me.